src/modules/template/service.js

```diff
@@ -1,8 +1,11 @@
 const { createTimestampHook } = require('@albert-team/mongol/builtins/hooks')
+const picomatch = require('picomatch')
 
 module.exports = class TemplateService {
   constructor(mongol) {
-    this.templateCollection = mongol.collection('template').attachHook(createTimestampHook())
+    this.templateCollection = mongol
+      .collection('template')
```

Comment:
```suggestion
      .collection('templates')
```
Số nhiều nhé, tương tự như "watches". Check lại xem còn chỗ nào dùng dạng số ít không sửa lại luôn nhé.